src/test/java/hudson/plugins/robot/RobotParserTest.java

```diff
@@ -183,6 +183,12 @@ public void testRobot4If() {
         final String dir = ".";
         final String mask = "robot4_if_output.xml";
         parse(dir, mask);
+    }
 
+    @Test
+    public void testRobot5Basic() {
```

Comment:
We will not remember in the future why we test Robot v.5 specifically. I would wish for more descriptive name (eg. `testRobot5TryAndWhile`) and a comment noting that this was introduced in RF5

Reply:
Renamed test and added a comment about test contents.